GoogleForCreators/web-stories-wp
# Add shortcut description for incremental nudge

## Context
This PR adds a description for `Shift + arrows` to the shortcuts menu.

## Summary

Before this PR there wasnt any description for `Shift + arrows` in shortcuts menu. This PR makes change to the shortcut menu by adding the required description. I have not added a test for this since there is already a test that covers this shortcut. (https://github.com/google/web-stories-wp/blob/aaa417091d9738d630fed4b8ac68eceb74167746/packages/story-editor/src/components/canvas/karma/keys.karma.js#L141-L155).

## Relevant Technical Choices
1. I could have used `⇧ + ↑ or ↓ or ←or →` but this choice would have messed up the UI. It would look somewhat like this
<img width="273" alt="Screenshot 2021-12-20 at 11 13 14 AM" src="https://user-images.githubusercontent.com/59614577/146717645-43f59732-a603-4a0d-adc6-9c56d36a2d9a.png">


## To-do



## User-facing changes
Before:
![image](https://user-images.githubusercontent.com/59614577/146716998-0b6b22ed-4bfe-4d2d-bfb3-316ac5ffacd7.png)
After:
![image](https://user-images.githubusercontent.com/59614577/146717029-95aea392-61ce-4c86-9a4e-04ac5d239aeb.png)


## Testing Instructions



This PR can be tested by following these steps:

1. Open the story-editor
2. Open the shortcuts menu.
3. Scroll down to the bottom of the shortcut menu
4. You should be able to see the new shortcut description.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---
Fixes #10015
